README.txt

```diff
@@ -48,6 +50,17 @@ from a previous version of this plugin, and you don't know how to fix the upgrad
 from the mod directory on your server and uninstall the plugin from system administration before trying again. 
 Note that this workaround will delete any instances of the plugin that you have set up in your courses.
 
+##Installation from Github
+
+This is recommended only for developers or if you need the very latest versions. 
+
+The plugin has two submodule dependencies and will not work directly from a clone. Go into the plugin folder
+and type the command...
+
+git submodule update --init --recursive
```

Comment:
Sorry, just saw this. No, that other stuff was deliberately deleted because its out of date.